core/src/cluster_info.rs

```diff
@@ -2492,7 +2491,7 @@ impl ClusterInfo {
                 let epoch = bank.epoch();
                 let epoch_schedule = bank.epoch_schedule();
                 epoch_time_ms = epoch_schedule.get_slots_in_epoch(epoch) * DEFAULT_MS_PER_SLOT;
-                staking_utils::staked_nodes(&bank)
+                bank.staked_nodes()
```

Comment:
Same here, rooted bank.

Reply:
again, i think would be better to make that change separately